lib/jobs/message_latency_report.ex

```diff
@@ -225,6 +227,20 @@ defmodule Jobs.MessageLatencyReport do
     |> round()
   end
 
+  defp record_log_files_size(log_files) do
+    Enum.reduce(log_files, 0, fn file, acc ->
```

Comment:
nit: We could `map` and `sum` instead of `reduce` here, if desired

Reply:
Will probably just keep it as a reduce since the difference is minimal in this case